scripts/proposals/coreGovProposalInterface.ts

```diff
@@ -1,6 +1,6 @@
 export interface CoreGovPropposal {
-    actionChainID: number;
-    actionAddress: string;
+    actionChainID: number | number[];
```

Comment:
what about instead an extension / override of CoreGovProposal (called e.g. CoreGovBatchProposal) and comment CoreGovProposal as deprecated?

Reply:
Agreed, it shouldnt really be possible to supply multiple actionChainIds and one action address. Ideally they would be the same number enforced by the interface